### PR TITLE
mohtashami2021reproducibility

### PR DESCRIPTION
**Pull request by mkrima**
Hello.

We added and reformatted our report. However, the font does not support bold mode. Is there anyway to fix this?

### CODE REVIEW

File: mohtashami2021reproducibility/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline mohtashami2021reproducibility/openreview/packages.tex:2]: Please remove the fontenc package, as discussed in README. Bold/italics will work with ReScience fonts when this package is removed from the import

Comment (mkrima) [reply mohtashami2021reproducibility/openreview/packages.tex]: Thank you very much. Done.

**Comment by koustuvsinha** [comment]
Thanks for the changes! LGTM!

@dodgejesse can you review the contents of the paper once?

**Review by dodgejesse — CHANGES_REQUESTED** [review]
Please include the first-page summary at the beginning of your submission. Because of the formatting differences you'll likely find that that first page summary actually goes onto the second page, and that there is then a page break before the intro, leaving most of the second page blank. You can remove this whitespace if you'd like, or leave it; up to you.

**Comment by mkrima** [comment]
I added the summary section to the beginning.

**Review by dodgejesse — APPROVED** [review]
Looks good to me! Nice work.

**Pull request merged**